app/src/main/res/drawable/ic_synced_history.xml

```diff
@@ -0,0 +1,14 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This icon should already exist in the codebase since it's used in the tabs tray - `ic_synced_tabs`. I would use that for now and we can look to see if we should be updating it as a followup.

Reply:
agreed, changed the existing to `ic_synced`